src/ReactiveUI.Wpf/TransitioningContentControl.cs

```diff
@@ -151,7 +152,7 @@ public enum TransitionDirection
         /// Gets or sets the transition duration.
         /// </summary>
         /// <value>The duration.</value>
-        public TimeSpan Duration { get => (TimeSpan)GetValue(TransitionDurationProperty); set => SetValue(TransitionDurationProperty, value); }
+        public TimeSpan Duration { get => (TimeSpan)GetValue(DurationProperty); set => SetValue(DurationProperty, value); }
```

Comment:
This is a breaking change and not keen changing it.

Reply:
@glennawatson ok